src/main/RemoteRequest.js

```diff
@@ -0,0 +1,87 @@
+/**
+ * RemoteRequest is used to for http requests on a remote server which can be
+ * fetched in chunks, e.g. using a Range request.
```

Comment:
Is this copy/paste from [`RemoteFile`][1]? I don't think it describes this class.

[1]: https://github.com/hammerlab/pileup.js/blob/55dc2213a9ccf6bdb336f402a42e0632776f8e7c/src/main/RemoteFile.js